lib/core/jtrace.c

```diff
@@ -383,7 +401,6 @@ j_trace_init (gchar const* name)
 	{
 		return;
 	}
-	else
```

Comment:
This should either not be removed or reformatted properly.

Reply:
The else was removed by an automated tool, because it has no effect after the "return" in the "true"-branch.
The code should be reformatted there, yes.